packages/augur-ui/src/modules/create-market/components/review.tsx

```diff
@@ -29,14 +27,14 @@ import { getCreateMarketBreakdown } from 'modules/contracts/actions/contractCall
 import {
   formatEtherEstimate,
   formatGasCostToEther,
-  formatPercent,
   formatDai,
-  formatEther
+  formatEther,
+  formatRep
 } from 'utils/format-number';
-import { Error } from 'modules/common/form';
+import { NoFundsErrors } from 'modules/create-market/components/no-funds-error'
```

Comment:
prob best to put this component in the create-market common folder. don’t need a new file for it.